src/test.js

```diff
@@ -204,6 +207,8 @@ Test.prototype = {
 			}
 		}
 
+		QUnit.warning( "testDone-duration" );
```

Comment:
Warnings shouldn't be generated when there's nothing the user can do about it. I'd like to see this appear only in a getter for `duration` where `Object.defineProperty` exists.